includes/Classifai/Providers/OpenAI/ChatGPT.php

```diff
@@ -141,6 +141,38 @@ public function register() {
 		add_action( 'enqueue_block_assets', [ $this, 'enqueue_editor_assets' ] );
 		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_admin_assets' ] );
 		add_action( 'edit_form_before_permalink', [ $this, 'register_generated_titles_template' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'admin_enqueue_script' ] );
+	}
+
+	/**
+	 * This function should enqueue the script for the admin page.
+	 *
+	 * @since x.x.x
```

Comment:
Next version will be 2.4.0 so we can set this:
```suggestion
	 * @since 2.4.0
```